src/main/java/ca/team2706/frc/robot/commands/drivebase/CurvatureDrive.java

```diff
@@ -47,13 +49,25 @@ public void initialize() {
 
     @Override
     public void execute() {
-        double rotation = (curveSpeed.get() > -0.05 && curveSpeed.get() < 0.05) ? 0 : curveSpeed.get();
-        boolean override = ((forwardVal.get() > -0.25 && forwardVal.get() < 0.25));
+        double forward = forwardVal.get();
+        double curve = curveSpeed.get();
+
+        if(squareInputs) {
+            // Uncomment for forward input squaring
+            //forward *= forward < 0 ? -forward : forward;
+            curve *= curve < 0 ? -curve : curve;
+        }
+
+        double rotation = (curve > -0.05 && curve < 0.05) ? 0 : curve;
```

Comment:
Might be beneficial to make a constant for this value of 0.05 deadband. Then you would do:
```java
Math.abs(curve) < 0.05
```

Reply:
We may also use this constant in OI too then.